MailCore/Models/NestableFolder.swift

```diff
@@ -31,6 +31,19 @@ public struct NestableFolder: Identifiable {
     /// `NestableFolder` childs
     public let children: [NestableFolder]
 
+    public var hasSubFolders: Bool {
+        let filteredChild = children.filter { $0.frozenContent.role == nil }
+        if !filteredChild.isEmpty {
+            return true
+        }
+        for child in children {
+            if child.hasSubFolders {
+                return true
+            }
+        }
+        return false
+    }
+
```

Comment:
I'm a bit worried about performance with the recursive algorithm :/
It would be nice to have @PhilippeWeidmann's opinion!

Maybe we could store and compute this value in Realm?

Reply:
Yep, we can do it in the background when fetching the folders `MailboxManager+Folders.swift:27`